psi4/src/read_options.cc

```diff
@@ -248,6 +248,40 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     /*- PCM-CCSD algorithm type. -*/
     options.add_str("PCM_CC_TYPE", "PTE", "PTE");
   }
+  
+  /*- PE boolean for polarizable embedding module -*/
+  options.add_bool("PE", false);
+  if (name == "PE" || options.read_globals()) {
+    /*- MODULEDESCRIPTION Performs polarizable embedding model (PE) computations. -*/
+
+    /*- Name of the potential file -*/
+    options.add_str_i("POTFILE", "potfile.pot");
+    /* - Use DIIS acceleration to obtain induced moments -*/
```

Comment:
Make sure the comment delimiters are exactly `/*-` and `-*/`. The scraper is picky